package.json

```diff
@@ -1,37 +1,65 @@
 {
   "name": "mustard",
   "version": "0.0.1",
-  "description": "A tagger tool for music files and simpe player",
+  "description": "A tagger tool for music files and simple player",
   "main": "main.js",
+  "homepage": "https://github.com/angrykoala/mustard#readme",
+  "dependencies": {
+    "browser-id3-writer": "^3.0.2",
+    "electron": "^1.4.14",
+    "jquery": "^3.1.1",
+    "musicmetadata": "^2.0.5",
+    "vue": "^2.1.8"
+  },
+  "devDependencies": {
+    "electron-packager": "^8.5.0",
+    "jshint": "^2.9.4"
+  },
   "scripts": {
     "start": "electron .",
     "test": "echo \"Error: no test specified\" && exit 1",
     "jshint": "jshint index.js app/*.js",
     "build-electron": "electron-packager . --overwrite --out builds --ignore='\\.gitignore|\\.jshintrc'",
```

Comment:
this may be no longer needed as you use electron builder instead

Reply:
I thought it was for testing purposes, although it isn't implemented yet.

Reply:
nope, that's why I'm telling you we don't need it anymore :smile:

Reply:
Ah ok